parsers/mysql/mysql.go

```diff
@@ -2,15 +2,20 @@
 package mysql
 
 import (
+	"fmt"
+	"log"
 	"regexp"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/Sirupsen/logrus"
+	"database/sql"
 
+	"github.com/Sirupsen/logrus"
+	_ "github.com/go-sql-driver/mysql"
 	"github.com/honeycombio/honeytail/event"
+	"github.com/percona/go-mysql/query"
```

Comment:
style nit - group the standard libs together, nonstandards together, and honeytail stuff together with 1 whitespace line between each (eg logrus, mysql should be together, honeytail/event on its own line, and database/sql with the other stdlibs)